docker-nginx/templates/default.conf.template

```diff
@@ -28,10 +28,17 @@ log_format json-logger escape=json
 
 
 upstream django {
+  # Defines a shared memory zone between worker processes, necessary for resolve keyword
```

Comment:
```suggestion
  # Defines a shared memory zone between worker processes, necessary for `resolver` keyword
```

I guess you mean the `resolver` here? I guess the 64k is an arbitrary number, shall we also mention it in the comment?

Reply:
The one second results in a docker internal dns lookup and is then shared between the workers for 1 second. This shouldn't have any performance impact.

When the app is not reachable, it fails after 1 second with either a 502 or 503 (I even saw a 500 error - it doesn't seem to consistent) which we then pickup to show a custom error message (see my newer commit).

Shared memory refers to the 'resolve' keyword that is reliant on it, clarified in a new commit.

Reply:
Thanks!